inst/htmljs/animint.js

```diff
@@ -1766,6 +1771,17 @@ var animint = function (to_select, json_file) {
       // event will be fired on the original input.
       selectized_array[v_name].setValue(selected_ids, true);
     }
+
+    // For each updated geom, check if the axes of the plot need to be
+    // updated and update them
+    s_info.update.forEach(function(g_name){
+      var plot_name = g_name.split("_").pop();
+      var axes = Plots[plot_name]["options"]["update_axes"];
+      if(axes != null){
+        update_scales(plot_name, g_name, axes, v_name, value);
```

Comment:
That is what I wanted to implement too at first, because its simpler. But there might be problems with that, since we are updating the scale itself. 

For example [in this example](http://bl.ocks.org/faizan-khan-iit/810a77ee29dcc110392305a80df86279) in the plot.json file under `axis_ranges` we have 2 geoms: `text` and `bar` (you could search for 'axis_ranges' in the page). Since we will be updating the scale for the plot every time before drawing the geom, the text may be rendered outside the boundaries of the plot. Click on Texas in the viz and you can barely see the state at the top. It will need to be fixed eventually so I thought we will have to update all the geoms.

Any ideas to implement this in a different manner?